docs/src/docs/asciidoc/getting-started.adoc

```diff
@@ -163,25 +167,29 @@ the configuration are described in the following listings:
 	}
 
 	asciidoctor { <6>
-		inputs.dir snippetsDir <7>
-		dependsOn test <8>
+		dependsOn test <7>
+		configurations 'asciidoctorExt'
+		baseDirFollowsSourceDir() <8>
 	}
 ----
 <1> Apply the Asciidoctor plugin.
-<2> Add a dependency on `spring-restdocs-asciidoctor` in the `asciidoctor` configuration.
-	This will automatically configure the `snippets` attribute for use in your `.adoc`
-	files to point to `build/generated-snippets`. It will also allow you to use the
-	`operation` block macro.
-<3> Add a dependency on `spring-restdocs-mockmvc` in the `testImplementation` configuration. If
+
+<2> Add a dependency on `spring-restdocs-mockmvc` in the `testImplementation` configuration. If
     you want to use `WebTestClient` or REST Assured rather than MockMvc, add a dependency
     on `spring-restdocs-webtestclient` or `spring-restdocs-restassured` respectively
     instead.
+<3> Add a dependency on `spring-restdocs-asciidoctor` in the `asciidoctorExt` configuration.
+	This will automatically configure the `snippets` attribute for use in your `.adoc`
+	files to point to `build/generated-snippets`. It will also allow you to use the
+	`operation` block macro.
 <4> Configure a property to define the output location for generated snippets.
 <5> Configure the `test` task to add the snippets directory as an output.
 <6> Configure the `asciidoctor` task
-<7> Configure the snippets directory as an input.
```

Comment:
Why has this been removed?

Reply:
Because according to the Gradle documentation, All files found under this snippetDir are treated as input files for this task. Is it what you want?

Reply:
See https://github.com/spring-projects/spring-restdocs/pull/744#discussion_r702912675.

Reply:
Done!